hammerdb/sql/drop-tables.sql

```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS customer;
+DROP TABLE IF EXISTS district;
+DROP TABLE IF EXISTS history;
+DROP TABLE IF EXISTS warehouse;
+DROP TABLE IF EXISTS stock;
+DROP TABLE IF EXISTS new_order;
+DROP TABLE IF EXISTS orders;
+DROP TABLE IF EXISTS order_line;
+DROP TABLE IF EXISTS item;
+
+-- ch benchmark tables
+DROP TABLE IF EXISTS region;
+DROP TABLE IF EXISTS nation;
+DROP TABLE IF EXISTS supplier;
```

Comment:
I sometimes get:
```sql
psql:drop-tables.sql:12: ERROR:  cannot drop table region because other objects depend on it                                                                                      
DETAIL:  constraint nation_n_regionkey_fkey on table nation depends on table region
HINT:  Use DROP ... CASCADE to drop the dependent objects too
```

This should maybe be:
```
DROP TABLE IF EXISTS supplier;
DROP TABLE IF EXISTS nation;
DROP TABLE IF EXISTS region;
```

Reply:
Maybe we can do:

```
DROP TABLE IF EXISTS supplier CASCADE;
DROP TABLE IF EXISTS nation CASCADE;
DROP TABLE IF EXISTS region CASCADE;
```
to be on the safe side?